html-training.md

```diff
@@ -0,0 +1,409 @@
+# HTML Training
+
+The next HTML snippets are examples of an accessible and SEO friendly semantic code, there are other ways to achieve the same results. These are based on the of the [WHATWG Living Standard](https://html.spec.whatwg.org/), best practices conventions and our dev experience dealing with UI hacks.
+
+This HTML guide is intended to be a quick reference on your development journey, for a deeper understanding we highly recommend [MDN Web Docs](https://developer.mozilla.org/).
+
+So you know at Z1 we encourage designers and developers to use native elements as they're meant to be. The Web is accessible by default, let's not break it!
+
+- [HTML Training](#html-training)
+  - [Links](#links)
+  - [Button](#button)
+  - [Lists](#lists)
+  - [Landmarks](#landmarks)
+    - [Banner](#banner)
+    - [Navigation](#navigation)
+    - [Main](#main)
+    - [Complementary](#complementary)
+    - [Contentinfo](#contentinfo)
+    - [Region](#region)
+  - [Navigation](#navigation-1)
+    - [Header](#header)
+    - [Breadcrumb](#breadcrumb)
+    - [Footer](#footer)
+  - [Cards](#cards)
+  - [Layout](#layout)
+    - [Page](#page)
+    - [Detail](#detail)
+
+## Links
+
+**Internal link**
+
+```html
+<a href="/about/">About us</a>
+```
+
+**Anchor links**
+
+```html
+<a href="#section-name">Section name</a>
+```
+
+```html
+<a href="#main-content" class="visually-hidden">Skip navigation</a>
+```
+
+**External links**
+
+```html
+<a href="https://blog.my-domain.com" rel="noopener noreferrer" target="_blank">
+  Our blog
+</a>
+```
+
+```html
+<a
+  href="https://marketing-partner.com"
+  rel="noopener noreferrer"
+  target="_blank"
+>
+  Marketing partner
+</a>
+```
+
+```html
+<a
+  href="https://github.com/"
+  rel="noopener noreferrer nofollow"
+  target="_blank"
+>
+  GitHub
+</a>
+```
+
+**Download link**
+
+```html
+<a href="./doc.pdf" download="file-name.pdf">Download doc</a>
+```
+
+**Image links**. If we wrap an image with a link this should have a `title` with information about the link not the image.
+
+```html
+<a href="/" title="Home">
+  <img src="./header-logo.png" alt="" />
+</a>
+```
+
+**Link with icon**. Any `svg` icon that is decorative or his meaning is redundant should have `aria-hidden` to no disturb the user experience and `focusable="none"` to prevent interact with this element.
+
+```html
+<a href="/post">
+  Next post
+  <svg aria-hidden="true" focusable="none"></svg>
+</a>
+```
+
+**Icon links**. If the link doesn't have a text we need to include it as `title`.
+
+```html
+<a href="#main-content" title="Back to top">
+  <svg aria-hidden="true" focusable="none" width="100" height="100">
+</svg>
+</a>
+```
+
+**Descriptive links**. The text should provide information about the link, if not we can add a `title` as description.
+
```

Comment:
Perhaps we could share a link to an article on the topic so you can better understand why this kind of copy ("click here", "learn more...") should be avoided and is the reason for the title attribute. 
This one could be interesting: https://www.smashingmagazine.com/2021/12/designing-better-links-websites-emails-guideline/